doc/user/content/known-limitations.md

```diff
@@ -25,6 +25,10 @@ tag](https://github.com/MaterializeInc/materialize/issues?q=is%3Aopen+is%3Aissue
 
 - `ANY` and `ALL` queries generate suboptimal plans {{% gh 744 %}}
 
+### Functions
+
+- Materialize does not support window functions. In some cases, you may be able to achieve the desired results with [temporal filters](https://materialize.com/temporal-filters/) or the [TOP K by group](../../sql/idioms/#top-k-by-group) idiom instead. {{% gh 213 %}}
```

Comment:
What do you think about linking “window functions” to https://twitter.com/xexd/status/1377599963161636865?s=21, to make it clear that a window function is something very specific?